e2e/suite.go

```diff
@@ -17,42 +17,27 @@ import (
 	"syscall"
```

Comment:
Perhaps some of this should be in a different PR, but I think it's OK to include now.

Reply:
@jscook2345 either way works for me and it is there, it is really up to you :)

Reply:
I am lazy, let's keep it